api/mess/views.py

```diff
@@ -85,7 +85,7 @@ def put(self, request, *args, **kwargs):
 
 
 class ComplaintView(APIView):
-    permission_classes = [IsAuthenticated]
+    permission_classes = [AllowAny]
```

Comment:
Why `AllowAny` is used here? Complaint should only be created by student.

Reply:
done